get_authors.rb

```diff
@@ -20,9 +52,9 @@ def main
   files = Dir.glob File.join('*', '**', '*.md')
 
   files.each do |f|
-    author_mail = get_authors(f).each_slice(2).to_h.sort
-
-    author_md = author_mail.collect { |a, e| Markdown.mailto(a, e) }
+    author_md = Git.log(f).author_email
+    author_md = author_md.sort
```

Comment:
Do you know, that we have `.sort!` exactly for that?
Instead, you can write just:
```
author_md.sort!
```

Reply:
We cannot use this, Array#sort! is not part of enumerable, maybe we could first transform it into array and then use sort!.

Reply:
But we have it for `Array` only. And since this is a `Hash` it performs an automatic conversion? What happens here?

Reply:
If sort is used on enum it gets transformed into Array. After quick glimpse at the ruby docs at [Enumerable#sort](http://ruby-doc.org/core-2.5.0/Enumerable.html#method-i-sort) we can see ```enum_to_a```  being used in source code, whereas at ```Array#sort!``` I couldn't spot anything similar..